deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,364 @@
+    mediatype: image/png   
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - nonResourceURLs:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: opendatahub-operator
+      deployments:
+      - name: opendatahub-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: opendatahub-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: opendatahub-operator
+            spec:
+              containers:
+              - command:
+                - kfctl
```

Comment:
How was the image built? Does it have kfctl or opendatahub-operator binary?

Reply:
Good catch. Tested and updated to run `opendatahub-operator`